src/test/java/net/masterthought/cucumber/generators/TagsOverviewPageTest.java

```diff
@@ -66,13 +66,14 @@ public void prepareReport_setTagsToExcludeFromChart_ReturnsFilteredTags() {
 
         // then
         VelocityContext context = page.context;
-        assertThat(context.get("chart_categories")).isEqualTo("[\"@fast\"]");
-        assertThat(context.get("chart_data")).isEqualTo(asList(
-                "[100.00]",
-                "[0.00]",
-                "[0.00]",
-                "[0.00]",
-                "[0.00]"));
+        assertThat(context.get("chart_categories")).isEqualTo(new String[]{"@fast"});
```

Comment:
there is probably method to extract element from that array and then compare the first one so `new String[]` is not needed any more